pom.xml

```diff
@@ -324,18 +324,6 @@
 				<version>2.0.1</version>
 			</dependency>
 
-			<!-- SLF4J Log bindings -->
-			<dependency>
-				<groupId>org.slf4j</groupId>
-				<artifactId>slf4j-api</artifactId>
-				<version>1.7.36</version>
-			</dependency>
-			<dependency>
-				<groupId>org.slf4j</groupId>
-				<artifactId>slf4j-jcl</artifactId>
```

Comment:
Are you sure `slf4j-jcl` is not used somewhere transitive and now will be using a different version?

Reply:
Theoretisch mogelijk maar ik zie deze nergens in de dependency-tree langskomen. Zal met de nieuwere versies van SLF4J en onze andere dependencies niet meer nodig zijn, lijkt het?

Reply:
We zetten alles om naar Logj42, dus ook slf4j, het is onzin om eerst naar jcl te gaan en dan naar log4j2? :)